packages/react/src/lib/accessibility/Behaviors/Toolbar/toolbarMenuItemBehavior.ts

```diff
@@ -0,0 +1,79 @@
+import { Accessibility, AccessibilityAttributes } from '../../types'
+import { IS_FOCUSABLE_ATTRIBUTE } from '../../FocusZone/focusUtilities'
+import * as keyboardKey from 'keyboard-key'
+// TODO: extend menuItemBehavior and ignore vertical
```

Comment:
Do we have a pattern of how we are extending other behaviors? This is basically the same as the menuItemBehavior, it's different only in a way that it does not have vertical prop (but behave as if it has as menus in toolbar are always vertical) @jurokapsiar @kolaps33